tests/large/test_eventgen_orchestration.py

```diff
@@ -282,7 +282,7 @@ def test_server_restart(self):
     def test_server_stop(self):
         r = requests.post("http://127.0.0.1:{}/stop".format(self.server_eventgen_webport))
         assert r.status_code == 200
-        assert "Eventgen is stopped" in r.content
+        assert "There is no Eventgen process running." in r.content
```

Comment:
Will `restart` operation restarts Eventgen process as previous configured?

Reply:
yes

Reply:
The the assertion should be `Eventgen is stopped` as previous test case restarts the server, right?